pandas/core/internals/blocks.py

```diff
@@ -925,26 +925,28 @@ def putmask(
         inplace: bool = False,
         axis: int = 0,
         transpose: bool = False,
-    ):
+    ) -> List["Block"]:
         """
         putmask the data to the block; it is possible that we may create a
         new dtype of block
 
-        return the resulting block(s)
+        Return the resulting blocks.
 
         Parameters
         ----------
-        mask  : the condition to respect
+        mask : the condition to respect
         new : a ndarray/object
-        align : boolean, perform alignment on other/cond, default is True
-        inplace : perform inplace modification, default is False
+        align : bool, default True
+            Perform alignment on other/cond.
+        inplace : bool, default False
+            Perform inplace modification.
         axis : int
-        transpose : boolean
-            Set to True if self is stored with axes reversed
+        transpose : bool, default False.
+            Set to True if self is stored with axes reversed.
 
         Returns
         -------
-        a list of new blocks, the result of the putmask
+        List[Block]
```

Comment:
from the feedback so far in #32773, it seems the majority view is to maintain a single standard for docstrings...

10 Errors found:
        Summary does not start with a capital letter
        Summary does not end with a period
        Summary should fit in a single line
        Parameter "mask" has no description
        Parameter "new" has no description
        Parameter "axis" has no description
        Parameter "transpose" type should not finish with "."
        Return value has no description
        See Also section not found
        No examples section found

Reply:
Having this as an example for the discussion in #32773 is really helpful.  I have opinions on this!  The capitalization and the punctuation I'm on board for changing, but I dont particularly want an Examples or See Also section

Reply:
> Having this as an example for the discussion in #32773 is really helpful.

i'll try not to disrupt your workflow too much 😉 

> but I dont particularly want an Examples or See Also section

This is the sort of thing that I was trying to convey with the term "lightweight", @jorisvandenbossche used the term "relaxing".

In numpydoc both these sections are optional. The pandas docstring guide does indicate that these sections are optional, but doesn't explicitly state that they are required either.